wsgi/odyssy/events/templates/events/event_list.html

```diff
@@ -0,0 +1,17 @@
+<body>
+    {% for event in events %}
+   <li><a href="{% url 'event-view-single' event.id %}">{{ event }}</a></li>
```

Comment:
Please be consistent with link address. Use this (with `url`), or the method you used in `wsgi/odyssy/basic/templates/basic/index.html` (with `/events/...`).